### PR TITLE
remove argument "ssrContext" from "createApp" function and it's subsequent calls

### LINKED ISSUE #10130: remove argument "ssrContext" from "createApp" function and it's subsequent calls
Based on the findings in https://github.com/rancher/dashboard/pull/9998

Remove argument `ssrContext` from `createApp` function and it's subsequent calls

### PR DESCRIPTION
**Pull request by aalves08**
### Summary
Fixes #10130 

### Occurred changes and/or fixed issues

- Remove argument `ssrContext` from `createApp` function and it's subsequent calls, since it was only used as `null`

### CODE REVIEW

File: shell/initialize/index.js [hunk 1]
```diff
@@ -174,29 +170,22 @@ async function createApp(ssrContext, config = {}) {
...
     error:           app.nuxt.error.bind(app),
-    payload:         ssrContext ? ssrContext.payload : undefined,
-    req:             ssrContext ? ssrContext.req : undefined,
-    res:             ssrContext ? ssrContext.res : undefined,
-    beforeRenderFns: ssrContext ? ssrContext.beforeRenderFns : undefined,
-    ssrContext
+    payload:         undefined,
+    req:             undefined,
+    res:             undefined,
+    beforeRenderFns: undefined
```
Comment (codyrancher) [inline shell/initialize/index.js:188]: Can these be removed?

File: shell/initialize/index.js [hunk 2]
```diff
@@ -174,29 +170,22 @@ async function createApp(ssrContext, config = {}) {
...
   await setContext(app, {
     store,
     route,
     next,
     error:           app.nuxt.error.bind(app),
-    payload:         ssrContext ? ssrContext.payload : undefined,
-    req:             ssrContext ? ssrContext.req : undefined,
-    res:             ssrContext ? ssrContext.res : undefined,
-    beforeRenderFns: ssrContext ? ssrContext.beforeRenderFns : undefined,
-    ssrContext
```
Comment (richard-cox) [inline shell/initialize/index.js:199]: i think this relates to `$ssrContext`, but couldn't prove as my instance died. once the pr that removes `fetch-server` merges could you take a look if other references can go to?

**Review by richard-cox — CHANGES_REQUESTED** [review]
Just one inline comment for possible work in this pr

File: shell/initialize/index.js (hunk 1 above)
Comment (aalves08) [reply shell/initialize/index.js:188]: I am unsure about `payload`, `req` and `res` but `beforeRenderFns` has no hits on the code so I assume it's safe to remove it.

File: shell/initialize/index.js (hunk 2 above)
Comment (aalves08) [reply shell/initialize/index.js:199]: Just removed all mentions to `ssrContext`. The remaining are on `fetch.server` which is about to be deleted on another PR.

Additionally, I removed another function `applySSRData` which wasn't doing anything at this point in time.

For another PR, we should search for all mentions to `ssr` on the codebase and check if they can be removed.

**Review by richard-cox — APPROVED** [review]

**Pull request merged**